$OpenBSD: patch-src_gmtk_media_player_c,v 1.14 2013/10/07 10:20:28 dcoppa Exp $

Add key bindings for h tv_step_channel down and k tv_step_channel
up (upstream svn revision r228)

--- src/gmtk_media_player.c.orig	Fri Aug 16 17:18:55 2013
+++ src/gmtk_media_player.c	Mon Oct  7 11:56:50 2013
@@ -834,6 +834,12 @@ static gboolean player_key_press_event_callback(GtkWid
         case GDK_o:
             write_to_mplayer(player, "osd\n");
             break;
+		case GDK_h:
+			write_to_mplayer(player, "tv_step_channel -1\n");
+			break;
+		case GDK_k:
+			write_to_mplayer(player, "tv_step_channel 1\n");
+			break;
         default:
             gm_log(player->debug, G_LOG_LEVEL_INFO, "ignoring key %s%s%s%s",
                    (event->state & GDK_CONTROL_MASK) ? "Control-" : "", (event->state & GDK_MOD1_MASK) ? "Alt-" : "",
